<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="de.uka.ilkd.key.parser.TestParallelParsing" tests="1" skipped="0" failures="1" errors="0" timestamp="2023-03-14T12:49:30" hostname="fv-az313-592" time="0.011">
  <properties/>
  <testcase name="testLoadingOfTwoDifferentProofFiles()" classname="de.uka.ilkd.key.parser.TestParallelParsing" time="0.011">
    <failure message="Could not initialize class de.uka.ilkd.key.nparser.KeYParser (file: src/test/resources/testcase/parser/MultipleRecursion/MultipleRecursion[MultipleRecursion__a()]_JML_normal_behavior_operation_contract_0.proof; caused by: java.lang.NoClassDefFoundError: Could not initialize class de.uka.ilkd.key.nparser.KeYParser)" type="de.uka.ilkd.key.proof.io.ProblemLoaderException">Could not initialize class de.uka.ilkd.key.nparser.KeYParser (file: src/test/resources/testcase/parser/MultipleRecursion/MultipleRecursion[MultipleRecursion__a()]_JML_normal_behavior_operation_contract_0.proof; caused by: java.lang.NoClassDefFoundError: Could not initialize class de.uka.ilkd.key.nparser.KeYParser)
	at app//de.uka.ilkd.key.control.AbstractUserInterfaceControl.load(AbstractUserInterfaceControl.java:221)
	at app//de.uka.ilkd.key.control.KeYEnvironment.load(KeYEnvironment.java:251)
	at app//de.uka.ilkd.key.control.KeYEnvironment.load(KeYEnvironment.java:224)
	at app//de.uka.ilkd.key.parser.TestParallelParsing$LoadThread.run(TestParallelParsing.java:113)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class de.uka.ilkd.key.nparser.KeYParser
	at de.uka.ilkd.key.nparser.ParsingFacade.createParser(ParsingFacade.java:87)
	at de.uka.ilkd.key.nparser.ParsingFacade.parseFile(ParsingFacade.java:123)
	at de.uka.ilkd.key.proof.io.KeYFile.getParseContext(KeYFile.java:160)
	at de.uka.ilkd.key.proof.io.KeYFile.getProblemInformation(KeYFile.java:236)
	at de.uka.ilkd.key.proof.init.KeYUserProblemFile.readProfileFromFile(KeYUserProblemFile.java:236)
	at de.uka.ilkd.key.proof.init.KeYUserProblemFile.getProfile(KeYUserProblemFile.java:216)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.createProblemInitializer(AbstractProblemLoader.java:508)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.loadEnvironment(AbstractProblemLoader.java:288)
	at de.uka.ilkd.key.proof.io.AbstractProblemLoader.load(AbstractProblemLoader.java:253)
	at de.uka.ilkd.key.control.AbstractUserInterfaceControl.load(AbstractUserInterfaceControl.java:209)
	... 3 more
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
